packages/shared/src/urbit/upload.ts

```diff
@@ -0,0 +1,161 @@
+import type { S3Client } from '@aws-sdk/client-s3';
```

Comment:
Wdyt about putting this file in `api`, rather than `urbit`?

Reply:
Yeah, good idea, just committed that.